docs/agents/agent-development.md

```diff
@@ -289,6 +308,21 @@ If a transaction is not sampled, you should set the `sampled: false` property an
 
 The agent should also have a sense of the most common libraries for these and instrument them without any further setup from the app developers.
 
+#### Span outcome
+
+The `outcome` property denotes whether the span represents a success or a failure.
+It supports the same values as `transaction.outcome`.
+The only semantic difference is that client errors set the `outcome` to `"failure"`.
+Agents should try to determine the outcome for spans created by auto instrumentation,
+which is especially important for exit spans.
```

Comment:
This is the first time the term "exit span" is used in our docs (if the github search is to be believed). Can you expand on what you mean with it?

/edit this should have been part of my review, somehow didn't work...

Reply:
Would you prefer this description, removing the term exit span explaining the term somewhere else in the spec?

```suggestion
which is especially important for exit spans (spans representing requests to other services).
```

Maybe for now just what I'm suggesting here ^ and extract to a common place once we use this term more often in the spec?

Reply:
I like the suggestion :+1: